jre_emul/android/platform/libcore/ojluni/src/main/java/java/util/jar/JarVerifier.java

```diff
@@ -146,6 +146,7 @@ public void beginEntry(JarEntry je, ManifestEntryVerifier mev)
                     return;
                 }
 
+                /* J2ObjC modified: copied from sun.misc.JarIndex */
                 if (uname.equals(JarFile.MANIFEST_NAME) ||
                         uname.equals(INDEX_NAME)) {
```

Comment:
Not quite. You already had `INDEX_NAME` declared with the comment that it's copied from sun.misc.JarIndex. Here, the comment is about how it diverges from the libcore source, which is `uname.equals(JarIndex.INDEX_NAME)`, and I think we should call out that we changed that line because we don't import `JarIndex` in this file.

Reply:
I see. I'll change it to "J2ObjC modified: platform-specific implementation to avoid importing sun.misc.JarIndex".

Reply:
LGTM